envs/example/defaults.yml

```diff
@@ -322,15 +323,15 @@ keystone:
     - name: heat
       type: orchestration
       description: 'Heat Orchestration API'
-      public_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      internal_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      admin_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
+      public_url: https://{{ endpoints.main }}:8004/v1/%(tenant_id)s
```

Comment:
We shouldn't be changing the ports here as part of this PR.   there is seperate work ( https://github.com/blueboxgroup/ursula/pull/1144 ) to adjust all of the ports to be correct in a way that it will not break existing ursula created stacks.